run_one_experiment.py

```diff
@@ -67,6 +67,7 @@ class AggregatedResult:
   max_coverage_sample: str = ''
   max_coverage_diff_sample: str = ''
   max_coverage_diff_report: str = ''
+  gained_textcov: Optional[textcov.Textcov] = None
```

Comment:
What does "gained_" here mean and how does it differ from "updated" in "textcov_updated"? we should be consistent with naming here, or just consider removing "gained_" and "_updated" completely and just calling these fields `coverage`